packages/lxd/src/query.coffee.md

```diff
@@ -36,16 +36,49 @@ Flags:
             The API path in the form of `[<remote>:]<API path>`, for example
             `/1.0/instances/c1`
             '''
+          'wait':
+            type: 'boolean'
+            default: false
+            description: '''
+            If true, activates the wait flag that waits for the operation to complete.
+            '''
+          'request':
+            enum: ['GET', 'PUT', 'DELETE', 'POST', 'PATCH']
+            default: 'GET'
+            description: '''
+            Action to use for the API call.
+            '''
+          'data':
+            type: 'string'
+            description: '''
+            Data to send to the action in the form of application/json stringified object.
+            '''
+          'parse':
+            type: 'boolean'
+            default: true
+            description: '''
+            Parse the response of the request as application/json.
+            ''' 
 
 ## Handler
 
     handler = ({config}) ->
       {stdout} = await @execute
-        command: [
-          'lxc', 'query', config.path
-        ].join ' '
+        command: """
+        #{[
+          'lxc', 'query', 
+          "--wait" if config.wait, 
+          "--request", config.request, 
+          "--data '#{config.data}'" if config.data?, 
+          config.path
+        ].join ' '}
+        """
```

Comment:
```suggestion
        command: [
          'lxc', 'query', 
          "--wait" if config.wait, 
          "--request", config.request, 
          "--data '#{config.data}'" if config.data?, 
          config.path
        ].join ' '
```

Do you really need to template the whole string from a `.join ' '` result ?

joining on an array already returns a string

Reply:
My bad. It was from the conversion js to coffee.